modules/javafx.graphics/src/test/java/test/javafx/scene/Node_lookup_Test.java

```diff
@@ -89,11 +96,40 @@ public class Node_lookup_Test {
     @Test public void lookupAllTest() {
         Set<Node> nodes = root.lookupAll("#a");
         assertEquals(1, nodes.size());
-        assertTrue(nodes.contains(a));
+        assertTrue(nodes.contains(ac));
 
         nodes = root.lookupAll(".d");
         assertEquals(2, nodes.size());
         assertTrue(nodes.contains(d));
         assertTrue(nodes.contains(d2));
     }
+
+    @Test
+    public void lookupPsuedoTest(){
```

Comment:
spelling: lookupPseudoTest
(also in the bug and PR title)

Reply:
Fixed the spelling issue in the class. However, as I have no permissions to edit the bug title, I cannot change the PR title till it is corrected by some one.

Reply:
@andy-goryachev-oracle  Thanks for correcting the bug title. Corrected the PR title now.